CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-- [...]
+- **[UPDATE]** Added rating prop to `TripCard`
```

Comment:
NEW ?

Reply:
Debatable, it's an update to TripCard, although it's updated with new stuff. I'd have used "UPDATE" myself

Reply:
I checked the changelog you're right NEW makes more sense based on on we used it before.